package.json

```diff
@@ -42,7 +42,7 @@
     "react-router": "^6.15.0",
     "react-router-dom": "^6.15.0",
     "react-select": "^5.7.4",
-    "recharts": "^2.8.0",
+    "recharts": "^2.13.0",
```

Comment:
Why was this updated? Is recharts even in use currently? I thought it was related to the currently disabled "timebank" views